Code/EnginePlugins/JoltPlugin/System/JoltQueries.cpp

```diff
@@ -327,14 +327,56 @@ void ezJoltWorldModule::QueryShapesInSphere(ezPhysicsOverlapResultArray& out_res
     return;
 
   const JPH::SphereShape shape(fSphereRadius);
```

Comment:
we should probably also multiply the radius here by ezMath::Max(vScale.x, vScale.y, vScale.z) to be consistent with the other two functions

Reply:
Maybe we should delegate scaling to Jolt? There is a shapeScale param, which seems to handle all cases well.
I have tested it for negative and zero scaling. Otherwise, in our implementation, there should be lots of `if condtions`.

Plus, I suppose, Jolt can handle non-uniform transform of spheres and capsules. Thus it should be possible to query ellipsoid-like objects.

Reply:
I have pushed the scaling through Jolt version. If you don't like it I will apply changes you proposed.